### ISSUE TITLE
Hardcoded path in the code makes 'backloop.dev' crash

### ISSUE THREAD

**Issue by bfontaine** [perki/backloop.dev#4]
Hello, and thank you for this very useful tool.

I’m using the NPM package and running a simple `backloop.dev .`, but as soon as I request https://whatever.backloop.dev:4443/, the server crashes because it tries to `mkdir` a [hardcoded path](https://github.com/perki/backloop.dev/blob/ec2021a6737fd08c76ac20eecdda4a2dc415d42f/nodejs/src/webserver/main.js#L68):
```
backloop.dev certificate OK, expires in 84 days
Server started on port 4443 serving files in '/home/baptiste/somedir'
You can open https://whatever.backloop.dev:4443/
/
node:fs:1372
  const result = binding.mkdir(
                         ^

Error: EACCES: permission denied, mkdir '/Users/perki/code/web-sherpy.ch'
    at Object.mkdirSync (node:fs:1372:26)
    at saveTofile (/home/baptiste/somedir/node_modules/backloop.dev/src/webserver/main.js:72:23)
    at fetchAndSave (/home/baptiste/somedir/node_modules/backloop.dev/src/webserver/main.js:82:9)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /home/baptiste/somedir/node_modules/backloop.dev/src/webserver/main.js:45:3 {
  errno: -13,
  code: 'EACCES',
  syscall: 'mkdir',
  path: '/Users/perki/code/web-sherpy.ch'
}

Node.js v20.16.0
```

Looking at the code, there’s another hardcoded string [here](https://github.com/perki/backloop.dev/blob/ec2021a6737fd08c76ac20eecdda4a2dc415d42f/nodejs/src/webserver/main.js#L78).

**Comment by bfontaine** [comment]
It seems that [this code](https://github.com/perki/backloop.dev/blob/ec2021a6737fd08c76ac20eecdda4a2dc415d42f/nodejs/src/webserver/main.js#L44-L45) shouldn’t be there; if I remove it everything works fine.

**Comment by perki** [comment]
@bfontaine Thanks a lot for reporting this issue.. it was trailing code I should have removed. 
I just published v2.0.4

**Comment by bfontaine** [comment]
Thank you; I confirm it works!
